packages/tpetra/core/example/BlockCrs/Tpetra_TestBlockCrs.cpp

```diff
@@ -471,7 +471,8 @@ int main (int argc, char *argv[])
       {
         TimeMonitor timerMultiVectorFill(*TimeMonitor::getNewTimer("4) MultiVectorFill"));
 
-        auto value = X->getLocalView<typename exec_space::memory_space>(Tpetra::Access::OverwriteAll);
+        using device_type = typename tpetra_multivector_type::node_type::device_type;
+        auto value = X->getLocalView<device_type>(Tpetra::Access::OverwriteAll);
```

Comment:
OK, but prefer use of getLocalViewDevice(Tpetra::Access::OverwriteAll).

Reply:
See [`3221242` (#9086)](https://github.com/trilinos/Trilinos/pull/9086/commits/3221242de7962a85ffddda5c3dc0c0c66f655540).